doc/source/howtos/proxysql.rst

```diff
@@ -65,33 +65,42 @@ Automatic Configuration
 =======================
 
 The ``proxysql`` package from Percona includes the ``proxysql-admin`` tool
-for configuring |PXC| nodes with ProxySQL::
+for configuring |PXC| nodes with ProxySQL.
```

Comment:
@rameshvs02 Can you review this part.

Reply:
@azhebel I have added few more updates in DOC-528. Could you please include those changes too.
PS : "--include-slaves" is not added in current commit

Reply:
ok